authentik/stages/authenticator_webauthn/tasks.py

```diff
@@ -27,9 +29,13 @@ def mds_ca() -> bytes:
         return _raw_root.read()
 
 
-@CELERY_APP.task()
-def webauthn_mds_import(force=False):
-    """Background task to import FIDO Alliance MDS blob into database"""
+@CELERY_APP.task(
+    bind=True,
+    base=SystemTask,
+)
+@prefill_task
```

Comment:
I don't remember if this needed to be in a specific order related to the celery decorator or not, but if this works then it's probably the correct order

Reply:
I picked it from somewhere else, so it should work